packages/server/integrations/jiraServer/JiraServerRestManager.ts

```diff
@@ -116,9 +117,7 @@ export default class JiraServerRestManager implements TaskIntegrationManager {
     const json = await response.json()
 
     if (response.status !== 201 && response.status !== 200) {
-      return new Error(
-        `Fetching projects failed with status ${response.status}, ${this.formatError(json)}`
```

Comment:
-1 why remove the description and `response.status`?

Reply:
@nickoferrall `Fetching projects failed` is an incorrect message, as it is not about projects, but an abstract error.
We also show the message directly on a frontend for `Jira`. e.g. incorrect JQL syntax. 
```
Error in the JQL Query: Expecting operator before the end of the query. The valid operators are '=', '!=', '<', '>', '<=', '>=', '~', '!~', 'IN', 'NOT IN', 'IS' and 'IS NOT'.
```
So I removed the status here as not required to display